cmd/account/mgmt/account-assign.go

```diff
@@ -97,10 +95,16 @@ func (o *accountAssignOptions) run() error {
 		rootID          string
 	)
 
-	if o.payerAccount == "osd-staging-1" {
+	const (
+		osdStaging1    = "osd-staging-1"
+		osdStaging2    = "osd-staging-2"
+		awsAccountName = "AWS_ACCOUNT_NAME"
+	)
```

Comment:
One _smalllll_ little nit here - we're redeclaring the consts in each of these files. Could we define these somewhere like in `cmd.go` and just reference them from all of these commands?